components/count-badge/README.md

```diff
@@ -62,28 +53,28 @@ The `d2l-count-badge-icon` element is a web component to display a number count,
-| `tab-stop`, default: `false` | Boolean | Optionally choose to make the badge a tab stop. |
-| `announce-changes`, default: `false` | Boolean | Optionally choose to announce changes to the badge with an aria-live region. If the text property is changed, the text will be read by screen-readers. |
-| `has-tooltip`, default: `false` | Boolean | Optionally choose to have a tooltip below the badge. |
-| `focus-ring`, default: `false` | Boolean | Optionally force a focus ring around the badge. This property can be used to highlight the badge when the parent element is focused. |
+| `number` | Number, required | The number to display on the badge.  Must be a positive integer. |
+| `icon` | String, required | [Preset icon key](../icons#preset-icons) (e.g. `tier1:gear`) |
+| `size` | String, default: `small` | The size of the badge. Valid options are `"small"` and `"large"`. |
+| `type` | String, default: `count` | The type of the badge. Valid options are `"notification"` and `"count"`. Notification badges are orange and count badges are grey. |
+| `max-digits` | Number, default: `2` when `type="notification"`, `5` when `type="count"` | Specifies a digit limit, after which numbers are truncated. Defaults to two for `"notification"` type and five for `"count"` type. Must be between 1-5.
+| `hide-zero` | Boolean, default: `false` | Hides the count badge when `number` is zero. |
+| `text` | String, required | Descriptive text for the badge which will act as an accessible label and tooltip text when tooltips are enabled. |
+| `tab-stop` | Boolean, default: `false` |  Adds a tab stop to the badge, which allows screen-reader users to easily tab to the badge. |
+| `announce-changes` | Boolean, default: `false` | Announces changes to the badge with an `aria-live` region. If the number property is changed, the text will be read by screenreaders. |
+| `has-tooltip` | Boolean, default: `false` | Adds a tooltip on the badge, which will be visible on hover/tab-stop, and read out by screen-readers. |
+| `focus-ring` | Boolean, default: `false` | Forces a focus ring around the badge. This property can be used to highlight the badge when the parent element is focused. |
 <!-- docs: end hidden content -->
+
+## Accessibility
+
+- While non-interactable components traditionally shouldn't be focusable, the `tab-stop` property greatly helps screen-reader users find the the count badge
+  - Otherwise, screen-reader users would have to use the arrow keys to get to the badge, which can take considerably longer
+  - If the optional tooltip is used, then `tab-stop` isn't necessary, as the count-badge becomes focusable to render the tooltip
+- The optional tooltip makes use of the [`d2l-toolip`](../../components/tooltip), which follows the [W3C best practices for Tooltips](https://www.w3.org/WAI/ARIA/apg/patterns/tooltip/)
+  - The use of this tooltip can help provide additional context such as providing the exact number, if it were to go above the limit set in `max-digits`
```

Comment:
Just trying to make it a bit more clear, though probably needs @geurts input on wording
```suggestion

Notable accessibility features:
- `tab-stop`: 
  - While non-interactable components traditionally shouldn't be focusable, this property greatly helps screen reader users find the the count badge, and removes the need to use the arrow keys to navigate to it.
  - If the optional tooltip is used, it isn't necessary, as the count-badge becomes focusable in order to display the tooltip
- `has-tooltip`: 
  - The optional tooltip makes use of the [`d2l-tooltip`](../../components/tooltip), which follows the [W3C best practices for Tooltips](https://www.w3.org/WAI/ARIA/apg/patterns/tooltip/)
  - This tooltip can help provide additional context, such as by providing the exact count badge number if it were to go above the limit set in `max-digits`
```

Reply:
While I do like some of the changes, I'm not sure about the formatting, so I'd also likely to hear Jeff's input

Reply:
Totally fair!